src/color/alert.json

```diff
@@ -2,172 +2,87 @@
   "color": {
     "alert": {
       "notification": {
-        "onLight": {
-          "value": "{color.brand.atlas.400.value}",
-          "public": false,
-          "onLight": true,
-          "usage": "Notification color on light backgrounds",
-          "wcag": "AAA",
-          "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "color-alert-notification-default",
-          "version": "3.15.0"
-        },
         "default": {
           "value": "{color.brand.atlas.400.value}",
-          "public": true,
+          "public": false,
```

Comment:
Why `false`?

Reply:
I made this `false` because when looking at deprecation structure that was in place before this work was started, the data was set to `"public": false`. 

Example: https://github.com/AlaskaAirlines/AuroDesignTokens/blob/5e5dce9085f294dda400adc1583a35566d30e735/src/color/ui.json#L7

Reply:
Ok. That's now how that works. Initial deprecation keeps this `true`. Its when we release a MAJOR version and we DO NOT want the tokens to be in the export we set that to `false`, but keep the token data in order to build the deprecation list data.

Reply:
```suggestion
          "public": true,
```